src/Document/Single.php

```diff
@@ -54,7 +55,7 @@ class Document extends Element
      *
      * @param mixed $id
      *
-     * @return Document
+     * @return $this
```

Comment:
PHPStorm and this page say otherwise: http://www.phpdoc.org/docs/latest/guides/types.html#keywords